tests/sanity.js

```diff
@@ -42,3 +42,11 @@ assert(new IntlPolyfill.DateTimeFormat('en', {
     month:'long',
     year: 'numeric'
 }).format(new Date('2016/01/16')), 'January 2016', 'month should be long');
+
+// Issue #192
+assert((function () {
```

Comment:
Thanks for adding the case. It would be nice if you can make the case like the others!
```js
assert(new IntlPolyfill.DateTimeFormat('en-us', {
    month:'long',
}).format(new Date(2016, 0, 1)), 'January', 'month should be long');
```